pkg/support/runsupportbundle.go

```diff
@@ -179,6 +140,21 @@ func (e *EksaDiagnosticBundle) PrintBundleConfig() error {
 	return nil
 }
 
+func (e *EksaDiagnosticBundle) WriteBundleConfig() error {
+	bundleYaml, err := yaml.Marshal(e.bundle)
+	if err != nil {
+		return fmt.Errorf("error outputing yaml: %v", err)
+	}
+	timestamp := time.Now().Format(time.RFC3339)
+	filename := fmt.Sprintf(generatedBundleNameFormat, e.clusterSpec.Name, timestamp)
+	e.bundlePath, err = e.writer.Write(filename, bundleYaml)
```

Comment:
so the bundle file will be written to CLUSTER_NAME dir? is a "support" dir necessary in this case?

Reply:
yup sounds good to me

Reply:
ok we're now writing the support bundle as a persistent file to `$CLUSTERNAME/support`

Reply:
so just to be clear, this writes the bundle config to $CLUSTERNAME/support, but the archived output/logs themselves are saved in $PWD